src/OpenIddict.EntityFramework/Models/OpenIddictApplication.cs

```diff
@@ -52,6 +59,11 @@ public class OpenIddictApplication<TKey> where TKey : IEquatable<TKey> {
         public virtual string Secret { get; set; }
 
         /// <summary>
+        /// Gets the list of the tokens associated with this application.
+        /// </summary>
+        public virtual IList<TToken> Tokens { get; } = new List<TToken>();
```

Comment:
We could, but we'd have to override `OpenIddictToken.Equals/GetHashCode()` for a hash set to be useful here. It's something Identity doesn't do, so not sure it's super useful.

Do you strongly feel we should do that?